### PR TITLE
add the possibility to add traincascade accuracy

### PR DESCRIPTION
**Pull request by StevenPuttemans**
Same as https://github.com/Itseez/opencv/pull/3798

### CODE REVIEW

**Comment by vpisarev** [comment]
:+1:

File: apps/traincascade/cascadeclassifier.cpp [hunk 1]
```diff
@@ -208,15 +210,20 @@ bool CvCascadeClassifier::train( const string _cascadeDirName,
...
             break;
         }
         if( tempLeafFARate <= requiredLeafFARate )
         {
             cout << "Required leaf false alarm rate achieved. "
-                 "Branch training terminated." << endl;
+                    "Branch training terminated." << endl;
             break;
         }
+        if( (tempLeafFARate <= acceptanceRatioBreakValue) && (acceptanceRatioBreakValue < 0) ){
```
Comment (vpisarev) [inline apps/traincascade/cascadeclassifier.cpp]: sorry, it should be "acceptanceRatioBreakValue &gt;= 0"

**Comment by StevenPuttemans** [comment]
Will fix it! And supply the master fix!

**Comment by StevenPuttemans** [comment]
Done!

**Comment by vpisarev** [comment]
:+1:

**Pull request merged**